modules/config_pr_bitbucket/src/RepoControllers/BitBucketController.php

```diff
@@ -3,130 +3,308 @@
 namespace Drupal\config_pr_bitbucket\RepoControllers;
 
 use Drupal\config_pr\RepoControllerInterface;
+use Bitbucket\API\Authentication;
+use Bitbucket\API\Http\Client;
+use Bitbucket\API\Repositories;
+use Bitbucket\API;
 
 /**
  * Class to define the BitBucket controller.
  *
  * @see \Drupal\config_pr\RepoControllerInterface
  */
-class BitBucketController implements RepoControllerInterface {
+class BitBucketController implements RepoControllerInterface
+{
 
   /**
    * Holds the controller name.
    *
-   * @var string $name.
+   * @var string $name .
```

Comment:
What is the space about

Reply:
I've removed them according to the drupal coding standards.